modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -135,11 +136,11 @@ public String getHelp() {
     @Override
     public void processOpts() {
         super.processOpts();
-        supportingFiles.add(
```

Comment:
If supporting files are disabled by config parameter, does it make sense to comment this out? Or do I understand this wrong?

Reply:
So I tried to use the config parameter `generateSupportingFiles` to disable supportingFiles, but it seems that for the typescript-angular stuff, people leverage the `supportingFiles` array to add some files specifically needed for the angular stuff.
Those files are always added regardless of the configuration.

Therefore, I needed to comment this out.
That is my understanding at least. Maybe I am missing something here.

Reply:
Okay, then maybe then remove the parameter to avoid confusion and leave a comment here that supporting files thing doesn't have effect, therefore we need to simple comment these lines?

Reply:
Comment is added. I hope now it makes sense.

Reply:
Parameter is removed as well. Sorry for the confusion.